www/guides/react.md

```diff
@@ -0,0 +1,373 @@
+---
+layout: layouts/guide.njk
+permalink: '/guides/react/'
+title: Snowpack + React Guide
+description: 'Documentation and guides for building projects with Snowpack and React.'
+date: 2020-10-01
+---
+
+Image: ideally a nice header image showing the before or after, or some illustration with elements of the Snowpack logo, React, and symbols related to speed, joy.
+
+Snowpack is a great fit for [React](https://reactjs.org/) projects of any size. It's easy to get started with and can scale to projects containing thousands of components and pages without any impact on development speed. Unlike traditional React application tooling, Snowpack saves you from getting bogged down with complex bundler setups and configuration files.
+
+In this guide, you'll go from a [minimal Snowpack project](https://github.com/pikapkg/snowpack/tree/master/create-snowpack-app/app-template-minimal) to a Snowpack project with React and several useful developer optimizations. In the process you'll learn:
+
+- How to set up a new Snowpack project
+- How to get started with React in a Snowpack project
+- Importing and using CSS and other assets inside your project
+- Enabling [Fast Refresh](https://reactnative.dev/docs/fast-refresh) mode for React
+- Extending Snowpack with plugins
+
+> If you want to jump to the end to see a full featured React setup, the Create Snowpack App React template comes with everything you'll learn in this tutorial plus other useful tools.
+
+Prerequisites: Snowpack is a command line tool installed from npm. This guide assumes a basic understanding of Node.js, npm, and how to run commands in the terminal. Knowledge of React is not required, Snowpack is a great way to learn React.
```

Comment:
okay I know we said no exclamation marks, but mannnnnnn I would add one at the end of that last sentence if I were writing that. Maybe: "Knowledge of React is not required. (Snowpack is a great way to learn React!)"?

this may just be a bad habit of mine :)

Reply:
We don't have to follow their standards, I'd say gonna just override here.